dist/index.html

```diff
@@ -1 +1,107 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Webpack Exercise</title><script defer="defer" src="main.js"></script></head><body><h1>Hello webpack</h1></body></html>
\ No newline at end of file
+<!DOCTYPE html>
```

Comment:
- [ ] OPTIONAL: I suggest you add some background to the comments section to improve it's readability:

![image](https://user-images.githubusercontent.com/57421823/153612050-def1cee4-6c29-43a4-8bcd-a619cc7eaacb.png)

- [ ] OPTIONAL: Every time I close the "reservations" popup, the page reloads.  You could find a way to prevent this, in order to improve the user experience.

Reply:
Resolved ✔️